src/troposphere_dns_certificate/certificate.py

```diff
@@ -95,13 +95,11 @@ def validate(e, p):
                     continue
 
                 if v['ValidationStatus'] == 'PENDING_VALIDATION':
-                    c = {}
```

Comment:
I had to change this structure back, since apparently the code right now is right on the verge of the 4096 byte limit.